### PR TITLE
Session and speaker pages

### PR DESCRIPTION
**Pull request by Snugug**
Pulling from live data in Sessionize, have the full-list session page and individual session pages.

* Individual session pages
* Session landing page to highlight just the sessions
* Schedule on homepage
* Homepage reworked to bring up CTAs and schedule

### CODE REVIEW

**Comment by webmaxru** [comment]
Homepage redesign:
- Focus on CTAs and Schedule now, pragmatic usage of the "first screen"
- Aligned to the brand colors
- Mobile view optimized

image

image

**Comment by Snugug** [comment]
Hey @webmaxru,

Can you please check with me before pushing code to a branch I'm working on? I'm going to need to roll this back because it doesn't align with the other design elements we have, breaks our build, and uses fixed desktop/mobile breakpoints which is not the responsive effect we're going for. There was a responsiveness issue below 470px which I missed, but instead of pushing code you should have mentioned it in code review.

**Comment by webmaxru** [comment]
Will comment on other points later but the build fails because of this Vite's bug/behavior(?): https://www.gitmemory.com/issue/vitejs/vite/4030/891728951 (empty `img src` attribute) - could you please remove "Marcos C" from the Panel on Sessionize (I believe this was a test entry and it's a duplicate and non-complete now)

**Comment by webmaxru** [comment]
@Snugug, I apologize for misunderstanding - I thought you have completed your work on scaffolding content, and now I can apply design (I told about this plan on Slack earlier). Will wait for this PR to be merged and create a next branch then. My points:
- Main intended change is a page layout. Now we have important info (Schedule) to show and we don't have the privilege to keep the "first screen" empty anymore
- I'm not sure which design elements do you mean: I fully align to our colors and use some elements from the static page, tickets page + some emphasis on the important CTAs. Furthermore, I plan to contribute the refreshed look from the main page back to all other pages (small tweaks to make everything consistent)
- We have fixed (you mean px?) breakpoints in multiple places in CSS, I follow exactly this idea. Let's chat separately about what do you want to change here
- Anyway, let's start with the build fix - correcting Sessionize data as I mentioned in the comment above should help here

File: src/js/lib/timezones.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    this.init();
+  }
+
+  /**
+   * Gets the user's stored timezone
+   */
+  async init() {
+    const offset = new Date().getTimezoneOffset();
+    this.local = `GMT${offset > 0 ? '-' : '+'}${Math.floor(offset / 60)
```
Comment (webmaxru) [inline src/js/lib/timezones.js:36]: It displays offset incorrectly: "GMT+-02.00". There is a fix:

```suggestion
    this.local = `GMT${offset > 0 ? '-' : '+'}${Math.abs(Math.floor(offset / 60))
```

File: src/js/lib/timezones.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      .toString()
+      .padStart(2, 0)}:${Math.floor(offset % 60)
```
Comment (webmaxru) [inline src/js/lib/timezones.js:38]: Same as above for XX:30 timezones, like Mumbai

```suggestion
      .padStart(2, 0)}:${Math.abs(Math.floor(offset % 60))
```

**Review by webmaxru — CHANGES_REQUESTED** [review]

**Review by webmaxru — APPROVED** [review]

**Pull request merged**